src/Domain/CoordinateMaps/KerrHorizon.hpp

```diff
@@ -12,24 +12,60 @@
 namespace domain::CoordinateMaps {
 
 /*!
- * Distorts cartesian coordinates \f$X^i\f$ such that a coordinate sphere
- * \f$\delta_{ij}X^iX^j=C\f$ is mapped to an ellipsoid of constant
- * Kerr-Schild-radius \f$r\f$, defined as the largest positive root of
+ *
+ * \brief Distorts cartesian coordinates \f$x^i\f$ such that a coordinate sphere
+ * \f$\delta_{ij}x^ix^j=C\f$ is mapped to an ellipsoid of constant
+ * Kerr-Schild-radius \f$r=C\f$.
+ *
+ * The largest Kerr-Schild radius \f$r\f$ is defined as the largest positive
+ * root of
  *
  *  \f{equation*}{
- *  r^4 - r^2 (\vec{x}^2 - \vec{a}^2) - (\vec{a}\cdot \vec{x})^2 = 0.
+ *  r^4 - r^2 (x^2 - a^2) - (\vec{a}\cdot \vec{x})^2 = 0.
  *  \f}
  *
- * In this equation, \f$\vec{x}\f$ is the coordinate vector to the (distorted)
+ * In this equation, \f$\vec{x}\f$ are the coordinates of the (distorted)
  * surface, and \f$\vec{a}=\vec{S}/M\f$ the spin-parameter of the black hole.
  * The black hole is assumed to be at the origin.
+ *
```

Comment:
Thanks for adding the documentation! One thing I'm confused on is whether `x` or `\xi` are the source coordinates. We generally use `\xi` as the source coordinates and `x` as the target coordinates. Could you make sure that's the notation being used here? We also use `\hat{\imath}` for the source coordinate indices (hats specifically)

Reply:
ok, so the `\hat{\imath}` just for the source coordinates? I added it, tell me if that is right, I cannot see this notation being used for the other coordinate maps.